code/modules/vehicles/mecha/combat/durand.dm

```diff
@@ -50,7 +50,7 @@
 
 /obj/vehicle/sealed/mecha/durand/process()
 	. = ..()
-	if(defense_mode && !use_energy(100 KILO JOULES)) //Defence mode can only be on with a occupant so we check if one of them can toggle it and toggle
+	if(defense_mode && !use_energy(100 JOULES)) //Defence mode can only be on with a occupant so we check if one of them can toggle it and toggle
```

Comment:
This should probably be rewritten in terms of `STANDARD_CELL_CHARGE`, or this bug could easily pop right back up.